wheel_thresh: [0, 0, 127.5, 255, 255, 191.25]
spigot_thresh: [0, 0, 0, 0, 0, 0]
shuttlecock_thresh: [0, 0, 0, 0, 0, 0]
switch_thresh: [0, 0, 0, 0, 0, 0]
wheel_thresh2: [0, 0, 0, 0, 0, 0]
spigot_thresh: [0, 0, 0, 0, 0, 0]
shuttlecock_thresh: [0, 0, 0, 0, 0, 0]
